src/resources/js/tribe-events-bar.js

```diff
@@ -393,3 +333,426 @@ var tribe_events_bar_action;
 
 })( window, document, jQuery, tribe_ev.data, tribe_ev.events, tribe_ev.fn, tribe_ev.state, tribe_ev.tests, tribe_debug );
 
+(function( factory ) {
```

Comment:
Rather than placing the Selectability plugin in `tribe-events-bar.js`, put it (and a minified version) in `vendor/selectability/`:

* `vendor/selectability/selectability.js`
* `vendor/selectability/selectability.min.js`